### PR TITLE
[TSD-158] Filter json link file

### PR DESCRIPTION
**Pull request by HirotoShioi**
YT: https://iohk.myjetbrains.com/youtrack/issue/TSD-158

### CODE REVIEW

**Comment by HirotoShioi** [comment]
Currently checking with logging-requirement if link file is included in the zip file.

**Comment by HirotoShioi** [comment]
Checked! We're ignoring `node.json` file.

File: classifier/src/Lib.hs [hunk 1]
```diff
@@ -467,34 +467,34 @@ showTickets tickets = do
 -- This function prints out the analysis result on the console.
 -- Can apply same refactoring as 'inspectAttachment'
 inspectLocalZipAttachment :: FilePath -> App ()
-inspectLocalZipAttachment filePath = do
+inspectLocalZipAttachment filePath =
+    flip catches [Handler handleZipFileException, Handler handleLogAnalysisException] $ do
```
Comment (ksaric) [inline classifier/src/Lib.hs]: `handle` - http://hackage.haskell.org/package/base-4.12.0.0/docs/Control-Exception.html#g:6 ?

File: classifier/src/Lib.hs [hunk 2]
```diff
@@ -467,34 +467,34 @@ showTickets tickets = do
...
-                    let errorCodes = extractErrorCodes analysisResult
+        printText "Analysis result:"
+        void $ mapM (printText . show) analysisResult
 
-                    printText "Analysis result:"
-                    void $ mapM (printText . show) analysisResult
+        printText "Error codes:"
+        void $ mapM printText errorCodes
+  where
+    handleZipFileException :: ZipFileExceptions -> App ()
```
Comment (ksaric) [inline classifier/src/Lib.hs]: Optional. You could have also used `TypeApplications` - `exceptionHandler @ZipFileExceptions` or `exceptionHandler @LogAnalysisException` to guide GHC.

**Review by ksaric — DISMISSED** [review]
LGTM, some optional changes.

**Review by ksaric — APPROVED** [review]
LGTM.

**Pull request merged**